Amazon.IonObjectMapper/Serializer/IonListSerializer.cs

```diff
@@ -33,27 +43,31 @@ public override System.Collections.IList Deserialize(IIonReader reader)
             IonType ionType;
             while ((ionType = reader.MoveNext()) != IonType.None)
             {
-                list.Add(serializer.Deserialize(reader, elementType, ionType));
+                list.Add(this.serializer.Deserialize(reader, this.elementType, ionType));
             }
+
             reader.StepOut();
-            
-            if (listType.IsArray)
+
+            if (this.listType.IsArray)
             {
-                var typedArray = Array.CreateInstance(elementType, list.Count);
-                for (int i=0; i<list.Count; i++)
+                var typedArray = Array.CreateInstance(this.elementType, list.Count);
+                for (int i = 0; i < list.Count; i++)
                 {
                     typedArray.SetValue(list[i], i);
                 }
+
                 return typedArray;
             }
-            
-            if (listType is System.Collections.IEnumerable || listType is object)
+
+            if (this.listType is System.Collections.IEnumerable || this.listType is object)
```

Comment:
Can we include `System.Collections` so we don't have to fully specify this namespace?

Reply:
And likewise for the other instances of `System.Collections` in this file.

Reply:
Done.